### PR TITLE
 Remove verbose flag from workflow and script

### PR DESCRIPTION
**Pull request by DanSava**
**Issue**

## Pre review checklist

- [x] Added appropriate release note label
- [x] PR title captures the intent of the changes, and is fitting for release notes.
- [x] Commit history is consistent and clean, in line with the [contribution guidelines](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md).
- [ ] Updated documentation
- [ ] Ensured new behaviour is tested

Adding labels helps the maintainers when writing release notes. This is the [list of release note labels](https://github.com/equinor/ert/labels?q=release-notes).

### CODE REVIEW

**Comment by DanSava** [comment]
Splitting up PR https://github.com/equinor/ert/pull/4744 into two parts.

File: src/ert/_c_wrappers/job_queue/workflow_job.py [hunk 1]
```diff
@@ -147,9 +147,7 @@ def run(self, ert: "EnKFMain", arguments: List[Any]) -> Any:
             self.__script = ExternalErtScript(ert, self.executable)
         else:
             raise UserWarning("Unknown script type!")
-        result = self.__script.initializeAndRun(
-            self.argumentTypes(), arguments
-        )
+        result = self.__script.initializeAndRun(self.argumentTypes(), arguments)
```
Comment (eivindjahren) [inline src/ert/_c_wrappers/job_queue/workflow_job.py:150]: This formatting change should probably be on the previous commit.

**Review by eivindjahren — APPROVED** [review]
Looks good! Commented on a minor detail that isn't so important.

File: src/ert/_c_wrappers/job_queue/workflow_job.py (hunk 1 above)
Comment (DanSava) [reply src/ert/_c_wrappers/job_queue/workflow_job.py:150]: Fixed

**Pull request merged**